src/explorer/model/Dependency.ts

```diff
@@ -11,7 +11,12 @@ const CONFLICT_INDICATOR: string = "omitted for conflict";
 export class Dependency extends TreeNode implements ITreeItem {
     private label: string = ""; // groupId:artifactId:version:scope
     private description: string = "";
-    constructor(value: string) {
+    private projectPomPath: string = "";
+    private groupId: string = "";
```

Comment:
declared but not used, as reported in Travis error messages.

Reply:
Do not initialize members as empty string if possible, you can perform null/undefined check later to avoid mistakes.